packages/components/src/stories/SquiggleChart.stories.mdx

```diff
@@ -79,6 +79,21 @@ could be continuous, discrete or mixed.
   </Story>
 </Canvas>
 
+### Date Distribution
+
+<Canvas>
+  <Story
+    name="Date Distribution"
+    args={{
+      code: "mx(10^12, 10^12 + 10^11, 10^12 + 2 * 10^11)",
```

Comment:
Is the idea that these mean milliseconds UNIX time or similar? We didn't discuss this. I tried to say that you give it the low-level distribution format, like, "{xs: [Date("...."), Date("")...], ys: [2.0, 4.0, 8.0]}" etc.

If you think that's too much work here we could do it as a different story later.

Reply:
ohh, doable.

Reply:
To be clear, this isn't asking for anything in Rescript. It's just that you get something to work for the Javascript record,
```
{
xs: [16610829770311, 1661829770311, 1661839770311, 1661849770311]
ys: [1.0, 2.0, 4.0, 1.0]
}
```

For example.

Sorry that wasn't clearer before.